internal/service/alarms/internal/db/migrations/000003_create_alarm_event_record_table.up.sql

```diff
@@ -10,8 +10,8 @@ CREATE SEQUENCE IF NOT EXISTS alarm_sequence_seq
 CREATE TABLE IF NOT EXISTS alarm_event_record (
     -- O-RAN
     alarm_event_record_id UUID PRIMARY KEY DEFAULT gen_random_uuid(), -- Unique identifier for each event record
-    alarm_definition_id UUID NOT NULL, -- From alarm_definition table
-    probable_cause_id UUID NOT NULL, -- From alarm_definition table
+    alarm_definition_id UUID, -- From alarm_definition table
```

Comment:
we've been trying to align this with the spec. These fields are not nullable according to O-CLOUD-IM.0-R004-v02.00